src/upload/system/library/transbank/utils/TransbankSdkWebpay.php

```diff
@@ -39,15 +40,11 @@ public function initTransaction($amount, $sessionId, $buyOrder, $returnUrl)
                     "url" => $response->url,
                     "token_ws" => $response->token
                 ];
-            } else {
-                throw new \Exception('No se ha creado la transacción para, amount: ' . $amount . ', sessionId: ' . $sessionId . ', buyOrder: ' . $buyOrder);
             }
-        } catch (\Exception $e) {
-            $result = [
-                "error" => 'Error al crear la transacción',
-                "detail" => $e->getMessage()
-            ];
-            $this->log->logError(json_encode($result));
+        }  catch (TransactionCreateException $e) {
+
+            $this->log->logError(json_encode($e->getMessage()));
+            return ["error" => $e->getMessage()];
```

Comment:
How to display in the front end this error?

Reply:
This is shown in the administrative panel when we make a connection test with Transbank, however a message appears in English and in the token section appears “undefined”, this will be recorded in the backlog to take it in a new task.